templates/multistage-buildconfig.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  generation: 1
+  name: jlinked-image
+  namespace: default
+spec:
+  lookupPolicy:
+    local: false
+status:
+  dockerImageRepository: image-registry.openshift-image-registry.svc:5000/default/jlinked-image
+  publicDockerImageRepository: default-route-openshift-image-registry.apps-crc.testing/default/jlinked-image
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata: 
+  name: jlinked-image
+spec:
+  source:
+    dockerfile: |
+	#Stage-1: ubi9-jlinked-image is builder image + application + jlinked JVM
```

Comment:
error parsing multistage-buildconfig.yaml: error converting YAML to JSON: yaml: line 8: found a tab character where an indentation space is expected